main.py

```diff
@@ -2,6 +2,42 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
+
+GENRES = [
+    "Western",
+    "Action",
+    "Dramma",
+]
+ACTORS = [
+    ("George", "Klooney"),
+    ("Kianu", "Reaves"),
+    ("Scarlett", "Keegan"),
+    ("Will", "Smith"),
+    ("Jaden", "Smith"),
+    ("Scarlett", "Johansson"),
+]
+
 
 def main() -> QuerySet:
-    pass
+    for genre in GENRES:
+        Genre.objects.create(name=genre)
```

Comment:
Genres and actors are used as global variables in your main function which is not a good practice - you either declare them inside main or pass to main() as arguments